new/PRS13997Valeriani.xml

```diff
@@ -0,0 +1,58 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS13997Valeriani" xml:lang="en" type="pers">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>D. Valeriani</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2023-04-18">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person>
+                    <persName xml:lang="gez" xml:id="n1">D. Valeriani</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1"/>
+                    <nationality type="Italy"/>
+                    <birth/>
+                    <death/>
+                    <floruit>19th century</floruit>
```

Comment:
see preface (Marrassini p. 83)

 Come curiosita si può anche notare la lunga
 lettera, non riportata per esteso nella descrizione del codice, annessa
 sempre all'Acq. e doni 38, in cui un D. Valeriani sfoggia coloritamente
 conoscenze non del tutto esatte sul contenuto del manoscritto; si tratta
 forse del Domenico Valeriani autore, fra l'altro, dell'Atlante del Basso
 e dell'Alto Egitto , stampato a Firenze nel 1837

Reply:
so we need at least addName (Prof), occupation (academic), reference to Florence, dates we know he was active in (1830, 1837) could also be encoded

Reply:
Like this: 
```
<person>
                    <persName xml:lang="it" xml:id="n1">
                        <addName>Prof</addName><forename>Domenico</forename><surname>Valeriani</surname>
                    </persName>
                    <nationality type="Italy"/>
                    <birth/>
                    <death/>
                    <occupation type="academic"></occupation>
                    <floruit notBefore="1830" notAfter="1837">mid-19th century in <placeName ref="LOC3096Floren"/></floruit>
                </person>
```